scen_edit/command/sync/team_sync.lua

```diff
@@ -82,11 +82,19 @@ function TeamManagerListenerWidget:onTeamAdded(teamID)
         return
     end
 
-    -- get local AI name (synced has weird AI names)
-    local aiID, _, _, name = Spring.GetAIInfo(teamID)
+    -- Setup the team names from unsynced side (Apparently those makes more
+    -- sense than synced ones).
+    local aiID, name, _, shortname = Spring.GetAIInfo(teamID)
     if aiID ~= nil then
-        team.name = tostring(teamID) .. ": " .. name
+        team.name = name
+    else
+        local _, leader, _, _, _, _ = Spring.GetTeamInfo(teamID)
```

Comment:
the `_`s after leader aren't necessary and should be removed (this happens in a few other places)